sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/transforms/GroupByKey.java

```diff
@@ -316,6 +317,13 @@ public void validate(PCollection<KV<K, V>> input) {
     return KvCoder.of(getKeyCoder(inputCoder), getOutputValueCoder(inputCoder));
   }
 
+  @Override
+  public void populateDisplayData(DisplayData.Builder builder) {
+    if (fewKeys) {
+      builder.add("fewKeys", true);
```

Comment:
May be worth making versions of the builder that allow omitting "default" values, and/or omitting null.

Eg, this could be:

builder.addWithDefault("fewKeys", fewKeys, false); // will add if fewKeys != the default value.